src/ResourceManager/Profile/Commands.Profile/help/Send-Feedback.md

```diff
@@ -38,7 +38,7 @@ Response.
 Please enter your email if you are interested in providing follow up information:
 
 your@email.com
-```
```

Comment:
@cormacpayne yeah that sounds fine. I just tested out generating maml and it works. the main issue was the example was missing the header. we can leave the back ticks as they were. Shall i close this PR and resubmit or what do u suggest?